clap_mangen/tests/testsuite/common.rs

```diff
@@ -324,6 +324,43 @@ pub(crate) fn value_name_without_arg(name: &'static str) -> clap::Command {
     )
 }
 
+
+pub(crate) fn configured_display_order_args(name: &'static str) -> clap::Command {
```

Comment:
Do we have sort order tests within `tests/builder`?  If so, seems like we should use that for easier comparison and to help catch things we might miss.